t/unit/transport/test_redis.py

```diff
@@ -733,12 +733,15 @@ def test_register_with_event_loop(self):
         transport.cycle = Mock(name='cycle')
         transport.cycle.fds = {12: 'LISTEN', 13: 'BRPOP'}
         conn = Mock(name='conn')
+        conn.client = Mock(name='client', transport_options={
```

Comment:
I think a dedicated test case for this feature (configurable health check interval) would be more extensible and easier to troubleshoot in the future. What do you think?

Reply:
As far as I can tell, three cases need to be examined:
- what happens if the connection class does not accept the `health_check_interval` argument?
- health check interval is configured and passed accordingly to `call_repeatedly` (already added, just needs to be extracted)
- health check interval is not configured (default value should be used)

Reply:
@georgepsarakis I added coverage for all the above.